src/test/java/divaeva/hw/api/store/GetStoreInventoryTest.java

```diff
@@ -0,0 +1,21 @@
+package divaeva.hw.api.store;
+
+import io.restassured.response.Response;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import rest.client.RestClient;
+
+import java.util.Map;
+
+public class GetStoreInventoryTest extends StoreTestBase {
+
+    private RestClient<Map> restClient = new RestClient<>(BASE_SERVER_URL_STORE_INVENTORY);
```

Comment:
I'm not sure that its a good idea for generic here. What if you need to work with different entities in one test?
And its a very strange use Map here for generic. Try to find another way for implementation

Reply:
fixed